go.mod

```diff
@@ -1,9 +1,8 @@
-module github.com/nishanths/license/v5
+module github.com/yuri-norwood/license/v5
```

Comment:
We should switch this back to the original module path?

Reply:
Oh sorry, I made that change to my local `main` branch so I could install my fork locally. I'll revert that.